src/Component/Header/Header.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import MainNavbar from './MainNavbar';
+import CATEGORY from './categoryData';
+import './Header.scss';
+
+class Header extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      menu: ['LOGIN', 'JOIN', 'CART', 'ORDER'],
+      category: [
+        'HEADWEAR',
+        'BAG',
+        'CLOTHES',
+        'SHOES',
+        'ACC',
+        'COMMUNITY',
+        'LOOKBOOK',
+      ],
+      categoryList: [],
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      categoryList: CATEGORY,
+    });
+  }
+
+  render() {
+    const { menu, category, categoryList } = this.state;
+
+    return (
+      <div className="headerContainer">
+        <div className="headerTop">
+          <div className="topWrapper">
+            <div className="headerLogo">
+              <a href="#" className="logoName">
+                GONGGOL
+              </a>
+            </div>
+            <ul className="headerMenu">
+              {menu.map((string, idx) => {
+                return (
+                  <li key={idx} className="listContainer">
+                    <a href="#">{string}</a>
+                  </li>
+                );
+              })}
+              <li>
+                <div className="dropDown">
+                  <button className="dropButton">
+                    MY PAGE
+                    <i class="fa fa-caret-down"></i>
+                  </button>
+                  <div className="dropContent">
+                    <a href="#">구매정보</a>
+                    <a href="#">배송조회</a>
+                    <a href="#">개인정보</a>
+                    <a href="#">나의리뷰</a>
+                    <a href="#" className="lastContent">
+                      1:1문의
+                    </a>
+                  </div>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="headerMiddle">
+          <div className="mainLogo">
+            <img src="../../../images/common/logo_tm.png" alt="메인로고" />
```

Comment:
```suggestion
            <img src="/images/common/logo_tm.png" alt="메인로고" />
```
이렇게 줄일 수 있습니다!
CRA 에서는 public 폴더 하위에 있는 정적파일들을 위와 같은 방식으로 접근할 수 있습니다! 
상대경로로 쉽게 가져올 수 있으니 이렇게 사용해주세요~